services/src/pro/datasets/external/sentinel_s2_l2a_cogs.rs

```diff
@@ -167,6 +167,7 @@ impl SentinelS2L2aCogsDataProvider {
                             no_data_value: band.no_data_value,
                         }
                         .into(),
+                        symbology: None, // TODO
```

Comment:
Can't you specify this 1-6000 or what this is, here?

Reply:
the colorizer is different for each band, so I can't easily specify it here

Reply:
We can use 0-10k and implement an if-else-block later.
Currently, you don't see anything which isn't a working solution.

https://gis.stackexchange.com/questions/233874/what-is-the-range-of-values-of-sentinel-2-level-2a-images

Reply:
caca41556981504247dd1879887e075b91eedc47